.travis.yml

```
@@ -8,6 +8,7 @@ env:
   matrix:
     - ACTION_MAILER_VERSION=4
     - ACTION_MAILER_VERSION=5
+    - ACTION_MAILER_VERSION=6.0.0.beta2
```

Once rails 6 is out, this should change as follows:

```suggestion
    - ACTION_MAILER_VERSION=6
```

---

I'll leave this as is but if you want me to update to beta3 to re-run CI to confirm, let me know. Happy to do it. Thanks!

---

Sure, thx!